### PR TITLE
[Bugfix] Replace the LF Support View

### PR DESCRIPTION
**Pull request by kcw-grunt**
### CRITICAL
The current UI causes users to accidentally send LTC to the LF Foundation
## Video
|Action Clip|
|----------|
|lfclipmove|

### CODE REVIEW

File: loafwallet/src/ModalPresenter.swift [hunk 1]
```diff
@@ -526,7 +533,27 @@ class ModalPresenter : Subscriber, Trackable {
...
         }
     }
+    // MARK: - Present Support LF View
+    private func presentSupportLF() {
+        
+        let supportLFView = UIHostingController(rootView: SupportLitecoinFoundationView(viewModel: SupportLitecoinFoundationViewModel()))
+            
+        supportLFView.rootView.viewModel.didCancel = {
+            supportLFView.dismiss(animated: true) {
+                //TODO: Track in Analytics
```
Comment (mosadialiou) [inline loafwallet/src/ModalPresenter.swift:543]: do you want to postpone the tracking?

File: loafwallet/src/ModalPresenter.swift [hunk 2]
```diff
@@ -526,7 +533,27 @@ class ModalPresenter : Subscriber, Trackable {
...
+            }
+        }
+        
+        supportLFView.rootView.viewModel.didCopyLTCAddress = {
+            supportLFView.dismiss(animated: true) {
+                //TODO: Track in Analytics
```
Comment (mosadialiou) [inline loafwallet/src/ModalPresenter.swift:549]: do you want to postpone the tracking?

**Review by mosadialiou — CHANGES_REQUESTED** [review]
Please attach a video clip

File: loafwallet/src/ModalPresenter.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/src/ModalPresenter.swift:543]: No...I need to do it right after. 
I need to get this to the App Store, ASAP.

File: loafwallet/src/ModalPresenter.swift (hunk 2 above)
Comment (kcw-grunt) [reply loafwallet/src/ModalPresenter.swift:549]: No...I need to do it right after. 
I need to get this to the App Store, ASAP.

**Pull request merged**